EventSubscriber/LogHistorySubscriber.php

```diff
@@ -206,15 +173,8 @@ private function getOriginalEntityData($entity) {
 
   /**
    * Find the new revision id for the current entity
-   *
-   * @param ClassMetadata $class
-   * @param               $entityData
-   * @param               $revType
-   *
-   * @return int|string
-   * @throws DBALException
    */
-  private function getRevisionId(ClassMetadata $class, $entityData, $revType) {
+  private function getRevisionId(ClassMetadata $class, $entityData, $revType): int|string {
```

Comment:
docblock seems to be wrong here...

```suggestion
  private function getRevisionId(ClassMetadata $class, $entityData, $revType): int{
```